scripts/DiffBranches.ps1

```diff
@@ -0,0 +1,76 @@
+[CmdletBinding()]
```

Comment:
It would be good to add some docs explaining how to use the script.
Ideally, something like in https://github.com/dotnet/extensions/blob/main/scripts/Slngen.ps1 or https://github.com/dotnet/extensions/blob/main/eng/scripts/ValidateProjectCoverage.ps1

Reply:
Thanks - added